common/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,111 @@
+  information regarding copyright ownership. The ASF licenses this file to
+  You under the Apache License, Version 2.0 (the "License"); you may not use
+  this file except in compliance with the License. You may obtain a copy of
+  the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required
+  by applicable law or agreed to in writing, software distributed under the
+  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+  OF ANY KIND, either express or implied. See the License for the specific
+  language governing permissions and limitations under the License. -->
+<configuration>
+
+  <if condition='property("drill.lilith.enable").equalsIgnoreCase("true")'>
+    <then>
+      <appender name="SOCKET" class="de.huxhorn.lilith.logback.appender.ClassicMultiplexSocketAppender">
+        <Compressing>true</Compressing>
+        <ReconnectionDelay>10000</ReconnectionDelay>
+        <IncludeCallerData>true</IncludeCallerData>
+        <RemoteHosts>${LILITH_HOSTNAME:-localhost}</RemoteHosts>
+      </appender>
+    </then>
+  </if>
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <!-- encoders are assigned the type
+         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <!--
+  <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+    <file>${log.path}</file>
+    <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
+      <fileNamePattern>${log.path}.%i</fileNamePattern>
+      <minIndex>1</minIndex>
+      <maxIndex>10</maxIndex>
+    </rollingPolicy>
+
+    <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
+      <maxFileSize>100MB</maxFileSize>
+    </triggeringPolicy>
+    <encoder>
+      <pattern>%date{ISO8601} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+  -->
+
+  <logger name="org.apache.drill" additivity="false">
+    <level value="debug"/>
+    <if condition='property("drill.lilith.enable").equalsIgnoreCase("true")'>
+      <then>
+        <appender-ref ref="SOCKET"/>
+      </then>
+    </if>
+<!--    <appender-ref ref="FILE" />-->
+  </logger>
+
+  <logger name="query.logger" additivity="false">
+    <level value="info"/>
+    <if condition='property("drill.lilith.enable").equalsIgnoreCase("true")'>
+      <then>
+        <appender-ref ref="SOCKET"/>
+      </then>
+    </if>
+  </logger>
+
+  <!--
+  <logger name="io.netty" additivity="false">
+    <level value="debug"/>
+    <if condition='property("drill.lilith.enable").equalsIgnoreCase("true")'>
+      <then>
+        <appender-ref ref="SOCKET"/>
+      </then>
+    </if>
+  </logger>
+  -->
+
+  <logger name="org.apache.hadoop" additivity="false">
+    <level value="info"/>
```

Comment:
See comment below about MapR. It is not clear that we always want info-level logging from Hadoop. Seems this should be enabled only in tests that want that detail to avoid cluttering output with unwanted messages.

Reply:
Thanks for pointing this, already removed.